katalog/node-exporter/deploy.yml

```diff
@@ -16,16 +16,19 @@ spec:
     spec:
       containers:
       - args:
-        - --web.listen-address=0.0.0.0:9100
```

Comment:
Why this was previously exposed on all interfaces?

Reply:
Because there was no `kube-rbac-proxy` so the `node_exporter` process needed to be reachable by Prometheus to be correctly scraped, now all the connections are authorized by `kube-rbac-proxy` which is bound to the node IP address and talks to `node_exporter` through the Pod's localhost.